spec/dummy/Gemfile.lock

```diff
@@ -4,7 +4,7 @@ PATH
     react_on_rails (1.0.0.pre)
       connection_pool
       execjs (~> 2.5)
-      rails (>= 4.0)
+      rails (>= 3.2)
```

Comment:
@robwise This shouldn't show up in your diff. It's a good practice to rebase your change on top of master, so long as nobody else is branching off of your branch.